src/class-admin.php

```diff
@@ -55,8 +57,17 @@ public function display_section() {
 	 * Display the setting field in the Reading settings page.
 	 */
 	public function display_setting() {
-		echo '<input name="joost_min_posts_count" id="joost_min_posts_count" type="number" min="1" value="' . esc_attr( \FewerTags\Plugin::$min_posts_count ) . '" class="small-text" /> ';
-		esc_html_e( 'posts before being live on the site.', 'fewer-tags' );
+		?>
+		<input
+			name="<?php echo esc_attr( Plugin::$option_name ); ?>"
+			id="<?php echo esc_attr( Plugin::$option_name ); ?>"
+			type="number"
+			min="1"
+			value="<?php echo esc_attr( Plugin::$min_posts_count ); ?>"
```

Comment:
Isn't this the wrong name?

Reply:
`name` and `id` were both set to `joost_min_posts_count`, so they were converted to `Plugin::$option_name`.
The `value` was set to `$min_posts_count` and was kept that way. Am I missing something? :thinking:

Reply:
Ah, ignore me, misreading.